worker/src/handlers/schedule.ts

```diff
@@ -69,18 +70,43 @@ async function resetQueue(sentry: Toucan): Promise<void> {
   }
   sentry.addBreadcrumb({ message: "Process complete" });
 }
+
 async function updateHistory(sentry: Toucan): Promise<void> {
   sentry.setTag("scheduled-task", "UPDATE_HISTORY");
   sentry.addBreadcrumb({ message: "Process started" });
   let data = createQueueData();
 
   sentry.addBreadcrumb({ message: "Get current data" });
   const analyticsData = await getAnalyticsData();
-  const timestampString = String(new Date().getTime());
+  const timestamp = new Date().getTime();
+  const timestampString = String(timestamp);
 
   sentry.addBreadcrumb({ message: "List UUID entries" });
   const kv_list = await listKV(KV_PREFIX_UUID);
 
+  const missingMetata = kv_list.filter((entry) => !entry.metadata);
+
+  async function handleMissingMetadata(entry: ListEntry) {
+    try {
+      const value = await KV.get<IncomingPayload>(entry.name, "json");
+      await KV.put(entry.name, JSON.stringify(value), {
+        expiration: entry.expiration,
+        metadata: generateUuidMetadata(value!, timestamp),
+      });
+    } catch (e) {
+      sentry.addBreadcrumb({ message: e });
+    }
+  }
+
+  if (missingMetata.length !== 0) {
+    await Promise.all(
```

Comment:
Log how many you're going to upgrade.

Reply:
Log? To where?

Reply:
To the log that we see if we fetch the logs via wrangler? Just so we can keep track of what happened if we see unclear data.

Reply:
That does not apply to ScheduleEvent

Reply:
`sentry.captureMessage` works

Reply:
`sentry.captureMessage` works